target/linux/ramips/dts/mt7621_dlink_dir-860l-b1.dts

```diff
@@ -114,6 +114,35 @@
 	};
 };
 
+&gmac0 {
+	mtd-mac-address = <&radio 0x4>;
```

Comment:
Where does this address come from? If it is correct, please change label-mac-device above to gmac0.
If it is not correct, label_mac in 02_network shouldn't be removed for this device.